pkg/datapath/linux/devices_controller_test.go

```diff
@@ -60,6 +60,17 @@ func devicesControllerTestSetup(t *testing.T) {
 	})
 }
 
+func containsAddress(dev *tables.Device, addrStr string) bool {
+	addr := netip.MustParseAddr(addrStr)
+	for _, a := range dev.Addrs {
+		if a.Addr == addr {
+			return true
+		}
+	}
+	return false
+
```

Comment:
extra empty line

Reply:
Since the tests passed and it's in a `_test.go` file I think I'll leave it. I'm definitely going to go back to these tests in the near future anyway.